docs/config-file-ort-yml.md

```diff
@@ -260,3 +260,57 @@ resolutions:
     reason: "INEFFECTIVE_VULNERABILITY"
     comment: "CVE-9999-9999 is a false positive"
 ```
+
+# License Choices
+
+### When to Use License Choices
+
+For multi-licensed dependencies a specific license can be selected.
+The license choice can be applied to a package.
+A choice is only valid for licenses combined with the SPDX operator `OR`.
+The choices are applied in the evaluator, and the reporter to the effective license of a package, which is calculated
+by the chosen LicenseView.
```

Comment:
Make "LicenseView" a link to the class so that readers can easily access the docs.